Core/CoreTiming.cpp

```diff
@@ -240,6 +240,11 @@ u64 GetTicks()
 	return (u64)globalTimer + slicelength - currentMIPS->downcount;
 }
 
+u64 GetTicksPerSecond()
```

Comment:
Removed, it was from initial porting which turned out to never needed to be used.